codingtest_ajw/2일차/수박수박수박수박수.js

```diff
@@ -0,0 +1,11 @@
+function solution(n) {
+    var answer = '';
+    for(let i=0;i<n;i++)
+    if(i%2==0)
+        answer+='수'
+    else
+        answer+='박'
+    return answer;
+}
```

Comment:
```suggestion
    let answer = '';
    
    for (let i=0; i<n; i++) {
      if (i % 2 == 0) {
          answer+= '수';
      } else {
          answer+= '박';
      }
    }
    
    return answer;
}
```
다른 피드백은 이전 내용에 있고 알아볼 수 있게 코드 컨벤션만 신경 써주시면 될 것 같습니다.